pytest.ini

```diff
@@ -0,0 +1,3 @@
+[pytest]
```

Comment:
no need for now since there is no new test involving an API_KEY (and they should not btw)